django_netjsonconfig/api/generics.py

```diff
@@ -1,7 +1,21 @@
-from rest_framework.generics import ListAPIView
+from rest_framework.generics import RetrieveAPIView, ListAPIView, get_object_or_404
 from rest_framework.response import Response
 
-from .serializers import SearchTemplateSerializer
+from .serializers import TemplateRetrieveSerializer, SearchTemplateSerializer
+
+
+class BaseGetTemplateView(RetrieveAPIView):
+    serializer_class = TemplateRetrieveSerializer
+
+    def get(self, request, *args, **kwargs):
+        key = request.GET.get('key', None)
+        if key:
+            temp = get_object_or_404(self.template_model, pk=kwargs['uuid'], key=key,
```

Comment:
let's avoid using `temp` as variable name, because it could mean "temporary", use `template` in this case, use `template`